synapse/config/server.py

```diff
@@ -143,6 +148,9 @@ def default_config(self, server_name, **kwargs):
         # Whether to serve a web client from the HTTP/HTTPS root resource.
         web_client: True
 
+        # The server's public-facing base URL
+        # https://example.com:8448/
```

Comment:
- Would prefer if this said it was explicitly for client-server API
- You forgot the the key name